bug fix - update-release-notes deletes existing release notes (#2408)
demisto/demisto-sdk: CHANGELOG.md

## Unreleased
* Fixed an issue where the **upload** failed on playbooks containing a value that starts with `=`.
* Fixed an issue where the **generate-unit-tests** failed to generate assertions, and generate unit tests when command names does not match method name.

> * Fixed an issue where the **update-release-notes** deleted the file when it already exists, instead of appending to it.

Reviewer: ```suggestion
* Fixed an issue where the **update-release-notes** deleted existing release notes, now appending to it instead.
```
